lib/init/grass.py

```diff
@@ -1900,10 +1900,21 @@ def csh_startup(location, location_name, mapset, grass_env_file):
     os.environ['HOME'] = userhome
     return process
 
+def shell_startup(location, location_name, grass_env_file, shell_name):
+    if shell_name not in ['bash', 'zsh']:
+        raise ValueError('Only bash and zsh are supported by shell_startup()')
+
+    if shell_name == 'zsh':
+        sh_history = ".zsh_history"
+        shrc = ".zshrc"
+        grass_shrc = ".grass.zshrc"
+    else:
+        sh_history = ".bash_history"
+        shrc = ".bashrc"
+        grass_shrc = ".grass.bashrc"
 
```

Comment:
May I suggest to keep the consistent order of `shell_name` here as the first `if shell_name` above? `bash` first, `zsh` next.